integration/apps/amg/0001-Insert-calls-to-geopm-profile-APIs-in-the-AMG-timing.patch

```diff
@@ -1,31 +1,19 @@
-From 312bdfcb681608d26b6b8a9824c4665144356647 Mon Sep 17 00:00:00 2001
-From: "Christopher M. Cantalupo" <christopher.m.cantalupo@intel.com>
-Date: Mon, 5 Oct 2020 13:41:31 -0700
-Subject: [PATCH 1/2] Insert calls to geopm profile APIs in the AMG timing.c
- file
+From ad6c399fd677b7a328261adb8fe69026562662a4 Mon Sep 17 00:00:00 2001
+From: Lowren Lawson <lowren.h.lawson@intel.com>
+Date: Wed, 4 May 2022 14:54:41 -0700
+Subject: [PATCH] Insert calls to geopm profile APIs in the AMG timing.c
 
 #  Copyright (c) 2015 - 2022, Intel Corporation
 #  SPDX-License-Identifier: BSD-3-Clause
 #
 
-Signed-off-by: Christopher M. Cantalupo <christopher.m.cantalupo@intel.com>
+Signed-off-by: Lowren Lawson <lowren.h.lawson@intel.com>
 ---
- .gitignore         |  2 ++
  Makefile.include   |  6 +++++-
- krylov/pcg.c       |  7 +++++++
- utilities/timing.c | 21 +++++++++++++++++++++
- 4 files changed, 35 insertions(+), 1 deletion(-)
- create mode 100644 .gitignore
+ krylov/pcg.c       |  8 ++++++++
+ utilities/timing.c | 22 ++++++++++++++++++++++
+ 3 files changed, 35 insertions(+), 1 deletion(-)
 
-diff --git a/.gitignore b/.gitignore
-new file mode 100644
-index 0000000..151a620
---- /dev/null
-+++ b/.gitignore
-@@ -0,0 +1,2 @@
-+*.o
-+*.a
```

Comment:
Why was the .gitignore removed?  This makes the build tree more clean to explore, especially when creating patch files.